lib/src/models/wallet_action/token_event.freezed.dart

```diff
@@ -323,7 +323,7 @@ class _$NativeTokenImpl extends NativeToken {
             @JsonKey(fromJson: nameFromJson) String name,
             @JsonKey(fromJson: addressFromJson) String address,
             BigInt value,
-            BigInt tokenId,
+            BigInt? tokenId,
```

Comment:
I see tokenId was addded to all the events, even to those that do not suppose to have it

Reply:
This is a generated file. We use the `tokenId` only in the `era721Transfer` union in `token_event.dart` file. Other union types do not have the `tokenId` field. @leonprou

Reply:
I know it's generated... one more issue that NFT transfer don't have value.
Anyway I don't have the tech expertise and context to make it right. 
I'm approving the PR if you think it's the best way to handle this

Reply:
I'll remove the value in a future PR. About having tokenId in multiple places, this is due to how freezed works. The tokenId is not added to events other than ERC-721. It has just been added to some functions that are required for some of the utilities that freezed offers like `maybeWhen`. Check out the code below for example. You won't see the `tokenId` field there.

```
abstract class NativeToken extends TokenEvent {
  const factory NativeToken(
          {final String symbol,
          final String name,
          final int decimals,
          final String address,
          @JsonKey(fromJson: amountFromJson) required final BigInt value}) =
      _$NativeTokenImpl;
  const NativeToken._() : super._();

  factory NativeToken.fromJson(Map<String, dynamic> json) =
      _$NativeTokenImpl.fromJson;

  @override
  String get symbol;
  @override
  String get name;
  int get decimals;
  @override
  String get address;
  @override
  @JsonKey(fromJson: amountFromJson)
  BigInt get value;
  @override
  @JsonKey(ignore: true)
  _$$NativeTokenImplCopyWith<_$NativeTokenImpl> get copyWith =>
      throw _privateConstructorUsedError;
}
```

Reply:
Ok @SametSahin10 , noted 👍